x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/spatial/search/aggregations/GeoShapeMetricAggregation.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.spatial.search.aggregations;
+
+import java.util.Map;
+
+public interface GeoShapeMetricAggregation {
+    Map<String, Object> geoJSON();
```

Comment:
Since the intention of this method is to return the underlying GeoJSON `Geometry` object, and the `InternalGeoLine`'s XContent object is the wrapping `Feature` object, I think it would make more sense to rename this to `geoJSONGeometry`.

Also, I think adding some javadoc here to explain its purpose would help a passerby in the future.